build/rust/Cargo.lock

```diff
@@ -2311,6 +2416,19 @@ dependencies = [
  "lazy_static",
 ]
 
+[[package]]
+name = "sharks"
+version = "0.5.0"
+source = "git+https://github.com/djandries/sta-rs?rev=7acfc2dfd12126f38dd81ea4734f37d6239d9605#7acfc2dfd12126f38dd81ea4734f37d6239d9605"
+dependencies = [
+ "bitvec",
+ "byteorder",
+ "ff 0.13.0",
```

Comment:
this is adding a second version of `ff`, can we use `0.12.1`? Also why is this in github.com/djandries and not brave?

Reply:
the lib PR was pending review, so i used my repo for the interim. will update shortly

Reply:
`ff` will stay at 0.13 due to breaking changes. still waiting for sta-rs merge, will probably update next week